homeassistant/components/sensor/alpha_vantage.py

```diff
@@ -89,6 +89,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     dev = []
     for symbol in symbols:
         try:
+            _LOGGER.debug('Configuring timeseries for symbols: %s',
```

Comment:
As per the [development guidelines](https://home-assistant.io/developers/development_guidelines/#quotes):

> Use single quotes `'` for single word and `"` for multiple words or sentences.